src/functions/Set-ItResult.ps1

```diff
@@ -67,60 +54,14 @@ function Set-ItResult {
     Assert-DescribeInProgress -CommandName Set-ItResult
 
     $result = $PSCmdlet.ParameterSetName
-    $message = "It result set to $result$(if ($Because) { ", $Because" })"
-    $data = @{
-        Result  = $result
-        Because = $Because
-    }
-    $errorRecord = New-PesterErrorRecord -Result $result  -Message $message -Data $data
-    throw $errorRecord
-}
 
-function New-PesterErrorRecord {
-    [CmdletBinding()]
-    param(
-        [Parameter(Mandatory = $true)]
-        [string]$Result,
-        [Parameter(Mandatory = $true)]
-        [string]$Message,
-        [string]$File,
-        [string]$Line,
-        [string]$LineText,
-        [hashtable]$Data
-    )
-
-    $exception = [Exception]$Message
-    $errorID = "PesterTest$Result"
-    $errorCategory = [Management.Automation.ErrorCategory]::InvalidResult
-
-    # we use ErrorRecord.TargetObject to pass structured information about the error to a reporting system.
-    $targetObject = @{
-        Message  = $Message
-        Data     = $Data
-        File     = $(if ($File -ne $null) {
-                $File
-            }
-            else {
-                $MyInvocation.ScriptName
-            })
-        Line     = $(if ($Line -ne $null) {
-                $Line
-            }
-            else {
-                $MyInvocation.ScriptLineNumber
-            })
-        LineText = $(if ($LineText -ne $null) {
-                $LineText
-            }
-            else {
-                $MyInvocation.Line
-            }).TrimEnd($([System.Environment]::NewLine))
+    #TODO: Remove in Pester 6
+    if ($result -in 'Inconclusive','Pending') {
```

Comment:
Yes please get rid if the warnings, these would trigger million times. We can add them back in when I make up my mind about the faith of Pending and Inconclusive.

Reply:
Removed.